dll/win32/comctl32/taskdialog.c

```diff
@@ -329,7 +329,7 @@ static unsigned int taskdialog_add_buttons(struct taskdialog_template_desc *desc
                 last_button = j;
 
         /* Difference in length of both lines if we wrapped the last button from the last line into this one */
-        diff_changed = abs(2 * buttons[last_button].width + line_widths[i] - line_widths[i - 1]);
+        diff_changed = 2 * buttons[last_button].width + line_widths[i] - line_widths[i - 1];
```

Comment:
Code change in Wine code: Surround the modification in `#ifdef __REACTOS__  ... #endif`

Reply:
Yes, will do that later.